contracts/RMRK/extension/erc20Holder/ERC20Holder.sol

```diff
@@ -19,32 +19,42 @@ abstract contract ERC20Holder is IERC20Holder {
         address erc20Contract,
         uint256 tokenId
     ) external view returns (uint256) {
-        return balances[tokenId][erc20Contract];
+        return _balances[tokenId][erc20Contract];
     }
 
     /**
-     * @inheritdoc IERC20Holder
+     * @notice Transfer ERC-20 tokens to a specific token
```

Comment:
I think these comments need to be changed as the function transfers the tokens from the token and not to it 🤔 

```suggestion
     * @notice Transfer ERC-20 tokens from a specific token.
```

Reply:
I think you are confused here. Since ERC20 has no onERC20Received method, this method will pull the tokens into the contract, to a specific token.

Reply:
All this time I was thinking these comments were `transferERC20ToToken`, sorry :grimacing: 
You were right on each of them